server/package.json

```diff
@@ -23,8 +24,30 @@
     "eslint-plugin-import": "^2.11.0",
     "eslint-plugin-node": "^6.0.1",
     "eslint-plugin-prettier": "^2.6.0",
+    "jest": "^22.4.3",
     "nodemon": "^1.17.3",
-    "prettier": "^1.12.1"
+    "prettier": "^1.12.1",
+    "supertest": "^3.1.0"
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "**/controllers/**/*.js",
+      "**/routes/**/*.js",
+      "**/etc/**/*.js",
+      "**/*.js",
```

Comment:
Does it make sense to whitelist three directories and then just include all `.js` files?

Reply:
Yes, I don't remember why include the `**/*.js`